pkgs/servers/monitoring/nagios-plugins/check_interfaces/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  check_interfaces,
+  fetchurl,
+  lib,
+  net-snmp,
+  nix-update-script,
+  stdenv,
+  testers,
+}:
+stdenv.mkDerivation rec {
+  pname = "check_interfaces";
+  version = "1.4.2";
+  src = fetchurl {
+    url = "https://github.com/NETWAYS/check_interfaces/releases/download/v${version}/check_interfaces-${version}.tar.gz";
+    hash = "sha256-w2CuJbFrYnC2JqCBCHxotxxZBHd9n3vU9ZFR9nQ/4I0=";
+  };
+
+  buildInputs = [ net-snmp ];
+
+  configureFlags = [ "--libexecdir=${placeholder "out"}/bin" ];
+
+  enableParallelBuilding = true;
+
+  postInstall = ''
+    # Remove unnecessary header files
+    rm --recursive $out/include
+  '';
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
+      package = check_interfaces;
+      # Program returns status code 3
+      command = "check_interfaces --help || true";
```

Comment:
which makes this check kinda redundant

Reply:
Would it be better for me to remove the check for now? I bet I can work with the upstream to support a proper version check command in the future.

Reply:
This has been changed to use the version command in the latest version, so it is no longer an issue.